lib/inventory_manipulation.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
```

Comment:
Shouldn't this file go in `lib/puppet_litmus/inventory_manipulation.rb` to follow Ruby module folder structure convention?